controllers/eventbus/installer/nats.go

```diff
@@ -414,7 +414,11 @@ func (i *natsInstaller) buildStanService() (*corev1.Service, error) {
 		Spec: corev1.ServiceSpec{
 			ClusterIP: corev1.ClusterIPNone,
 			Ports: []corev1.ServicePort{
-				{Name: "client", Port: clientPort},
+				// Prefix tcp- to enable clients to connect from
+				// an istio-enabled namespace, following:
+				// https://github.com/nats-io/nats-operator/issues/88
+				// https://github.com/istio/istio/issues/28623
+				{Name: "tcp-client", Port: clientPort},
```

Comment:
Have you tested if this will impact upgrading from an existing EventBus?

Reply:
Hi!

Good point, I need to test with and without Istio (not just with). I will have time in the weekend to do more thoroughly testing, any particular scenario you would like to see checked?

- Without Istio, run pre-patch, apply post-patch, everything should still work.
- With Istio, run pre-patch (doesn't work), apply post-patch, everything should work.
- Fresh installs on both cases.

Does this cover it?

Reply:
Sorry for the late response, just came back from vacation.

I'm more concerned about if the port name change is allowed on an existing service.

Reply:
Got it! I'll check that as well, some delays getting time on my side.

Reply:
I tested it, upgrading has no issue, thanks!

Reply:
Awesome! Thank you very much! I was unable to get time for this, deployments "season" at work have kept me busy and I kept postponing it :/

Thanks again!